streams/src/test/java/org/apache/kafka/test/MockProcessorContext.java

```diff
@@ -121,6 +121,14 @@ public RecordCollector recordCollector() {
         return recordCollector;
     }
 
+    public void setKeySerde(final Serde<?> keySerde) {
```

Comment:
Why do we add this? Seems to be unused?

Reply:
It is added to be symmetry with `setValueSerde`.